pep-0690.rst

```diff
@@ -547,17 +606,15 @@ issue, since imports inside a ``with`` block are always eager.
 Deferred Exceptions
 -------------------
 
-All exceptions arising from import (including ``ModuleNotFoundError``) are
-deferred from import time to first-use time, which could complicate debugging.
-Referencing a name in the middle of any code could trigger a deferred import and
-produce ``LazyImportError`` while loading and executing the related imported
-module.
+Exceptions that occur during a lazy import bubble up and erase the
+partially-constructed module(s) from ``sys.modules``, just as exceptions during
+normal import do.
 
-Ensuring all lazy import errors are raised as ``LazyImportError`` mitigates this
-issue by reducing the likelihood that they will be accidentally caught and
-mistaken for a different expected exception.  ``LazyImportError`` will also
-provide the location of the original import statement to aid in debugging, as
-described above.
+Since errors raised during a lazy import will occur later (wherever the imported
```

Comment:
I think this would read better if the parenthetical statement came later, such as "...if the import were eager (i.e. whenever the name is first reference), it is also possible ...".  Note the "i.e."